cloudlift/deployment/service_template_generator.py

```diff
@@ -254,7 +258,7 @@ def _add_service(self, service_name, config):
                 )
             ]
         if 'logging' not in config or 'logging' in config and config['logging'] is not None:
-            container_definition_arguments['LogConfiguration'] = self._gen_log_config(service_name, "awslogs" if 'logging' not in config else config['logging'])
+            container_definition_arguments['LogConfiguration'] = self._gen_log_config(service_name, "awsfirelens" if 'logging' not in config else config['logging'])
```

Comment:
Default needs to be aws logs

Reply:
default logging has been set to `awslogs` for now